htdocs/core/class/dolreceiptprinter.class.php

```diff
@@ -633,6 +635,7 @@ public function sendToPrinter($object, $templateid, $printerid)
 	{
 		global $mysoc, $langs, $user;
 		global $hookmanager;
+		$hookmanager->initHooks(array('receiptPrinter', 'globalcard'));
```

Comment:
The context of a hook must be initialized into the calling main page.

Reply:
This is really a mistake as the calling page contains a context. So, I abandon the pull request. Thanks.